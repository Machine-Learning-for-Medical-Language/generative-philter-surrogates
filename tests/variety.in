A two word name: Abigail Spencer.
A really long name: Abigail Bianca-Twist Spencer-Smith
With a prefix: Mrs Bianca Twist
A short name: Bob

Age: 31 years old
Date: 2021/3/21
Date2: 2021-03-21

Address: 216 Main St.
Full address: 216 Main St Apt 3, Austin, TX
State: to California.
State and city name: Newton, TX.
State and city name2: Somerville, Massachusetts

Phone: 555-234-1532
Email: abigail@gmail.com
Url: testing.com/here

Identifier thing: X13451N
SSN: 516-27-1455
SSN-like: 1516-27-1455
Random number: 152346

Hospital1: brought to Saint Bartholemew
Hospital2: admitted from Newbury Nursing Center
Hospital3: chilling at Newbury Nursing Center