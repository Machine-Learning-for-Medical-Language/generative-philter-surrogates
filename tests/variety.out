A two word name: Shawn Rodriguez.
A really long name: Heather Kruegerheatherkruege-James
With a prefix: Mrs Charles Webb
K short name: Amy

Age: 31 years old
Date: 1991-03-1
Date2: 2008-08-02

Address: 5611 Smith .
Full address: 715 Hardy Walks 3, Lake Tanya
State: Mi East Laura.
State and city name: New Robert.
State and city name2: Levytonlevytonlevytonlevy

Phone: 363-908-9073
Email: hpark@example.org
Url: http://hall.com/

Identifier thing: 6685509
SSN: 371983-07-1
SSN-like: 5262008-06-0
Random number: 603330

Hospital1: brought to Odonnell and Sons
Hospital2: admitted from Brandon Nursing Center
Hospital3: chilling Gardner, Butler and Parks
